.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -10,13 +10,12 @@ Closes #
 -
 
 ### Checklist
-- [ ] My code follows the [code-style](https://github.com/GIScience/ohsome-api/blob/master/CONTRIBUTING.md) rules, and I have checked on the [static analyses](https://jenkins.ohsome.org/job/oshdb/view/change-requests/) and [benchmark](https://reports.ohsome.org/oshdb-benchmarks/) (if applicable) results
+- [ ] My code follows the [code-style](https://github.com/GIScience/ohsome-api/blob/master/CONTRIBUTING.md) rules, and I have checked on the [static analyses](https://jenkins.ohsome.org/job/ohsome-api/view/change-requests/)
 - [ ] I have commented my code
 - [ ] I have written javadoc (required for public methods)
 - [ ] I have added sufficient unit tests
```

Comment:
Should we add additionally “add API tests”?

Reply:
Some mentioning of tests should be given in the template yes, so the contributor gets reminded and has to think about tests.

Reply:
So would you suggest something like this?
```suggestion
- [ ] I have added sufficient unit and API tests
```
or
```suggestion
- [ ] I have added sufficient unit tests
- [ ] I have added sufficient API or integration tests
```

Reply:
I'd say one line mentioning the tests is enough, so the upper version.